editor/components/block-toolbar/index.js

```diff
@@ -1,87 +1,38 @@
 /**
  * External dependencies
  */
-import classnames from 'classnames';
 import { connect } from 'react-redux';
 
 /**
  * WordPress Dependencies
  */
-import { IconButton, Toolbar, Slot } from '@wordpress/components';
+import { Slot } from '@wordpress/components';
 import { Component } from '@wordpress/element';
-import { __ } from '@wordpress/i18n';
 
 /**
  * Internal Dependencies
  */
 import './style.scss';
 import BlockSwitcher from '../block-switcher';
-import BlockMover from '../block-mover';
-import BlockInspectorButton from '../block-settings-menu/block-inspector-button';
-import BlockModeToggle from '../block-settings-menu/block-mode-toggle';
-import BlockDeleteButton from '../block-settings-menu/block-delete-button';
-import UnknownConverter from '../block-settings-menu/unknown-converter';
 import { getBlockMode, getSelectedBlock } from '../../store/selectors';
 
 class BlockToolbar extends Component {
```

Comment:
This could be transformed to a functional component I guess?

Reply:
Not entirely sure how to do that, feel free to push if you have the energy.